readme.md

```diff
@@ -82,6 +82,10 @@ The following are **feature flags** that can be used with `--preset` flag (they
     <td style="vertical-align: middle;">--detox</td>
     <td style="vertical-align: middle;">Generate workflow to run Detox e2e tests on every PR</td>
   </tr>
+  <tr>
+    <td style="vertical-align: middle;">--maestro</td>
+    <td style="vertical-align: middle;">Generate workflow to run Maestro e2e tests on every PR</td>
```

Comment:
I wonder if we shouldn't extend this readme (or create some extended version somewhere else (np next to each recipe) and write what it does under the hood in more details + known issues like the one with iOS

Reply:
This will be done as part of https://github.com/software-mansion-labs/react-native-ci-cli/issues/72